### DISCUSSION TITLE
Welcome to rustle Discussions!

### DISCUSSION THREAD

**Discussion by pintariching** [pintariching/rustle#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by linrium** [comment]
Hi, I'm Backend Engineer working on Rust and Go, but a long time ago I was a Frontend Enginner, so I still love to use Rust to enhance the JS tool. I found this repo on Reddit and thought it so great. I'm learning Rust and have a small Telegram group that help newbie can learn Rust easier. Feel free to join :")
